app/src/main/java/com/example/myapplication/mvp/model/CalculatorModel.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.myapplication.mvp.model
+
+import com.example.myapplication.utils.EMPTY_STRING
+import com.example.myapplication.utils.NULL_RESULT
+
+class CalculatorModel {
```

Comment:
here you can use a data class
https://www.baeldung.com/kotlin-data-classes

Reply:
I think that is not the best idea because the model not ever will have only 4 variables with their setters and getters, in my opinion, is fine just like this